### ISSUE TITLE
make install doesn't install files

### ISSUE THREAD

**Issue by mxmilkiib** [ahlstromcj/seq66#7]
This PKGBUILD completes without error, but it also doesn't actually install any files to the pkgdir to get compressed and archived;

```
_pkgname=seq66
pkgname="${_pkgname}-git"
pkgver=0.90.4.r186.f4f1619
pkgrel=1
pkgdesc="A live-looping sequencer with an interface similar to hardware sequencers"
arch=('i686' 'x86_64')
url="https://github.com/ahlstromcj/seq66"
license=('GPL2')
depends=('jack' 'qt5-base' 'gambas3-gb-qt5-opengl')
makedepends=('git')
groups=('pro-audio')
provides=("${_pkgname}" "${_pkgname}=${pkgver//.r*/}")
conflicts=("${_pkgname}")
source=("${_pkgname}::git+https://github.com/ahlstromcj/${_pkgname}.git#branch=optimizing")
md5sums=('SKIP')

pkgver() {
  cd "${srcdir}/${_pkgname}"

  local ver=$(tail -n 1 VERSION)
  # replace alpha chars with a dot
  echo "$ver.r$(git rev-list --count HEAD).$(git rev-parse --short HEAD)"
}

build() {
  cd "${srcdir}/${_pkgname}"
  qmake -makefile -recursive "CONFIG += rtmidi" seq66.pro
  make
}

package() {
  cd "${srcdir}/${_pkgname}"
  make DESTDIR="${pkgdir}/" install
}
```

Edit: here's the output when running;
```
...
==> Starting package()...
cd libseq66/ && ( test -e Makefile || /usr/bin/qmake -o Makefile /home/milk/state/pkgs/mine/seq66-git/src/seq66/libseq66/libseq66.pro 'CONFIG += rtmidi' ) && make -f Makefile install
make[1]: Entering directory '/home/milk/state/pkgs/mine/seq66-git/src/seq66/libseq66'
make[1]: Nothing to be done for 'install'.
make[1]: Leaving directory '/home/milk/state/pkgs/mine/seq66-git/src/seq66/libseq66'
cd libsessions/ && ( test -e Makefile || /usr/bin/qmake -o Makefile /home/milk/state/pkgs/mine/seq66-git/src/seq66/libsessions/libsessions.pro 'CONFIG += rtmidi' ) && make -f Makefile install
make[1]: Entering directory '/home/milk/state/pkgs/mine/seq66-git/src/seq66/libsessions'
make[1]: Nothing to be done for 'install'.
make[1]: Leaving directory '/home/milk/state/pkgs/mine/seq66-git/src/seq66/libsessions'
cd seq_rtmidi/ && ( test -e Makefile || /usr/bin/qmake -o Makefile /home/milk/state/pkgs/mine/seq66-git/src/seq66/seq_rtmidi/seq_rtmidi.pro 'CONFIG += rtmidi' ) && make -f Makefile install
make[1]: Entering directory '/home/milk/state/pkgs/mine/seq66-git/src/seq66/seq_rtmidi'
make[1]: Nothing to be done for 'install'.
make[1]: Leaving directory '/home/milk/state/pkgs/mine/seq66-git/src/seq66/seq_rtmidi'
cd seq_qt5/ && ( test -e Makefile || /usr/bin/qmake -o Makefile /home/milk/state/pkgs/mine/seq66-git/src/seq66/seq_qt5/seq_qt5.pro 'CONFIG += rtmidi' ) && make -f Makefile install
make[1]: Entering directory '/home/milk/state/pkgs/mine/seq66-git/src/seq66/seq_qt5'
make[1]: Nothing to be done for 'install'.
make[1]: Leaving directory '/home/milk/state/pkgs/mine/seq66-git/src/seq66/seq_qt5'
cd Seq66qt5/ && ( test -e Makefile || /usr/bin/qmake -o Makefile /home/milk/state/pkgs/mine/seq66-git/src/seq66/Seq66qt5/Seq66qt5.pro 'CONFIG += rtmidi' ) && make -f Makefile install
make[1]: Entering directory '/home/milk/state/pkgs/mine/seq66-git/src/seq66/Seq66qt5'
make[1]: Nothing to be done for 'install'.
make[1]: Leaving directory '/home/milk/state/pkgs/mine/seq66-git/src/seq66/Seq66qt5'
==> Tidying install...
...
```

**Comment by mxmilkiib** [comment]
Right, to the problem with the [original PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=seq66-git) was that my system also had gcc8 installed, after removing that, it worked.

Maybe I'm missing the point of the shadow build method and it is just for testing?

**Comment by ahlstromcj** [comment]
The shadow build for seq66 is just a thing that qmake provides, and it is also
used when using Qt Creator to build the project.  I did beef up the qmake build
so that I could also build seq66 using the rtmidi engine (the default is the
less functional portmidi engine that is used when building for Windows/Mac).
The reason I added rtmidi to the qmake process is so that I could build for
debugging.  For some reason (gcc 9?) when I build in debug mode using automake,
I get a link error about a missing static function (building in release mode is
fine).  This error doesn't happen when building using qmake.  Not sure why.

That's on my Debian Sid laptop.  On my Ubuntu laptop, no problem.

-------- Milkii Brewster 16:21 Thu 09 Jul      --------

>Right, to the problem with the [original PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=seq66-git) was that my system also had gcc8 installed, after removing that, it worked.
>
>Maybe I'm missing the point of the shadow build method and it is just for testing?
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/ahlstromcj/seq66/issues/7#issuecomment-656397072

-- 
It's now the GNU Emacs of all terminal emulators.
		-- Linus Torvalds, regarding the fact that Linux started off as a terminal emulator

**Comment by mxmilkiib** [comment]
After providing config.log to folk in #archlinux-proaudio yesterday;
```
<strogon14_> why do you have g++-8 ?
<nilshi> retro compilation, adds more warmth to the sound
<strogon14_> yeah, makes the MIDI signals more analog
<milkii> strogon14_: not sure, maybe trying to build something that wasn't updated at some point
<strogon14_> milkii: I could image that that's what makes the compilation fail on line 1015 of config.log
<strogon14_> It's strange that configure throws away the stdout and stderr of check commands. Also, I seem to remember in other projects, when a check fails, config.log has the C source code of the check that failed. Not so here. Why?
```

I had thought any build system would choose the latest available in the local system.

Is accounting for the scenario I had possible? Or should this issue be closed?

**Comment by ahlstromcj** [comment]
What is this stuff about retro compilation adds more warmth to the sound?  Seems like a vinyl-vs-CD issue to me!  :-)

**Comment by ahlstromcj** [comment]
The Debian Sid build issue seems to have been fixed in 0.91.0 by adding --copy-dt-needed-entries to the Makefile.  The other issues seem to be upstream, so will close this one out.
